device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/port_config.ini

```diff
@@ -1,35 +1,33 @@
-# name          lanes             alias         index
-Ethernet0       1,2,3,4           Ethernet1/1    1
-Ethernet4       5,6,7,8           Ethernet2/1    2
-Ethernet8       9,10,11,12        Ethernet3/1    3
-Ethernet12      13,14,15,16       Ethernet4/1    4
-Ethernet16      21,22,23,24       Ethernet5/1    5
-Ethernet20      17,18,19,20       Ethernet6/1    6
-Ethernet24      25,26,27,28       Ethernet7/1    7
-Ethernet28      29,30,31,32       Ethernet8/1    8
-Ethernet32      37,38,39,40       Ethernet9/1    9
-Ethernet36      33,34,35,36       Ethernet10/1  10
-Ethernet40      41,42,43,44       Ethernet11/1  11
-Ethernet44      45,46,47,48       Ethernet12/1  12
-Ethernet48      53,54,55,56       Ethernet13/1  13
-Ethernet52      49,50,51,52       Ethernet14/1  14
-Ethernet56      57,58,59,60       Ethernet15/1  15
-Ethernet60      61,62,63,64       Ethernet16/1  16
-Ethernet64      69,70,71,72       Ethernet17/1  17
-Ethernet68      65,66,67,68       Ethernet18/1  18
-Ethernet72      73,74,75,76       Ethernet19/1  19
-Ethernet76      77,78,79,80       Ethernet20/1  20
-Ethernet80      85,86,87,88       Ethernet21/1  21
-Ethernet84      81,82,83,84       Ethernet22/1  22
-Ethernet88      89,90,91,92       Ethernet23/1  23
-Ethernet92      93,94,95,96       Ethernet24/1  24
-Ethernet96      101,102,103,104   Ethernet25/1  25
-Ethernet100     97,98,99,100      Ethernet26/1  26
-Ethernet104     105,106,107,108   Ethernet27/1  27
-Ethernet108     109,110,111,112   Ethernet28/1  28
-Ethernet112     117,118,119,120   Ethernet29/1  29
-Ethernet116     113,114,115,116   Ethernet30/1  30
-Ethernet120     121,122,123,124   Ethernet31/1  31
-Ethernet124     125,126,127,128   Ethernet32/1  32
-Ethernet128     129               Ethernet33    33
```

Comment:
Does it help if we add Ethernet128, Ethernet129 with speed 10G in port-config.ini? I mean why remove altogether?

Reply:
I tried with updating just the speed and not removing the ports and the issue is not seen anymore. There is another issue, however - the original port_config.ini had duplicated lane 128 on two ports.
We can add 10G ports back again when @Staphylo confirms what should be the correct lane allocation.